R/observations.R

```diff
@@ -75,12 +83,15 @@ get_observations <- function(streamid,
                 })
             rm(req)
             rm(response)
+            if (name_streams) {
```

Comment:
It might not be necessary to name a stream and show warning message if you just retrieve a single stream. As it will be easy to rename column name outside rsenaps.